packages/caliper-samples/src/contract/scalardl/src/main/java/com/example/contract/simple/SimpleOpen.java

```diff
@@ -0,0 +1,26 @@
+package com.example.contract.simple;
+
+import com.scalar.ledger.asset.Asset;
+import com.scalar.ledger.contract.Contract;
+import com.scalar.ledger.ledger.Ledger;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+
+public class SimpleOpen extends Contract {
+
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> properties) {
+    String accountId = argument.getString("account");
```

Comment:
It's more appropriate to check if the argument contains the `account` key otherwise it will throw NPE.
Please check it out.
https://github.com/scalar-labs/scalardl/blob/master/docs/how-to-write-contract.md

Reply:
Done including other contracts.